### PR TITLE
adding nuget formula

### PR DESCRIPTION
**Pull request by blairham**

### CODE REVIEW

File: Library/Formula/nuget.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Nuget < Formula
+  desc "NuGet is the package manager for the Microsoft development platform including .NET"
+  homepage "https://wwww.nuget.org/"
```
Comment (mistydemeo) [inline Library/Formula/nuget.rb]: Typo?

File: Library/Formula/nuget.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  url "https://s3.amazonaws.com/homebrew-nuget/nuget-3.2.0.10516.tar.gz"
```
Comment (mistydemeo) [inline Library/Formula/nuget.rb]: What distinguishes the homebrew package from the others?

File: Library/Formula/nuget.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  depends_on "mono"
+
+  def install
+    bin.install Dir["*"]
+  end
+
+  test do
+    system "#{bin}/nuget"
+  end
+end
```
Comment (mistydemeo) [inline Library/Formula/nuget.rb]: Newline at end of file, please.

File: Library/Formula/nuget.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "1a76e705a3253d020c5ee8a92816f9c10b8604226eab7fcd196ae29e8bfacbda"
+
+  depends_on "mono"
+
+  def install
+    bin.install Dir["*"]
+  end
+
+  test do
+    system "#{bin}/nuget"
```
Comment (mistydemeo) [inline Library/Formula/nuget.rb]: Is there any actual functionality you can test?

**Comment by DomT4** [comment]
Any update on an answer to Misty's question? Thanks!

> What distinguishes the homebrew package from the others?

**Comment by blairham** [comment]
I'm not 100% sure how to answer this, but I will attempt. This package enables you to use NuGet on OSX. It mainly distributes Nuget.exe along with a bash script that passes it through mono.

**Comment by mikemcquaid** [comment]
> I'm not 100% sure how to answer this, but I will attempt. This package enables you to use NuGet on OSX. It mainly distributes Nuget.exe along with a bash script that passes it through mono.

We need to build this from source instead.

**Comment by blairham** [comment]
Good luck with that it's a windows app.

**Comment by mikemcquaid** [comment]
@blairham The homebrew/binary tap and Homebrew Cask may accept it. It's unclear to me that this is an official release supported by `nuget`, though so I'd be concerned about support issues.

**Comment by blairham** [comment]
Ok I'll look into that, this was mainly a way to distribute the bash script to make it work on OS X. I'll also look and see if the code will compile on mono, I have serious doubts it will but I'm willing to attempt some day.

**Comment by mikemcquaid** [comment]
@blairham This is the type of thing where unless the Bash script is trivial enough to be included in the formula (e.g. <10 lines) then we'd like to see upstream distribute it and provide official support. Otherwise we're not really installing `nuget` but `nuget with a Bash wrapper that someone wrote".

**Comment by blairham** [comment]
Understood, then this will never fly as the whole point of the bash script is to make it work on OS X as it just wraps it around mono. I'll just drop it from here

**Comment by darkn3rd** [comment]
http://www.codedistillers.com/?p=99

**Pull request closed without merging**